windows/src/desktop/kmshell/main/UfrmOnlineUpdateIcon.pas

```diff
@@ -123,7 +123,7 @@ procedure TfrmOnlineUpdateIcon.trayClick(Sender: TObject);
 begin
   tmrShowBalloon.Enabled := False;
   tray.Visible := False;
-  with TOnlineUpdateCheck.Create(True, False) do
+  with TOnlineUpdateCheck.Create(nil, True, False) do
```

Comment:
Why is this `nil` instead of `Self`?

Reply:
Because `Self` here is an invisible window, so making it owner doesn't really work.